### PR TITLE
Cleanup: Migration plan options

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.4

Partially resolves https://issues.redhat.com/browse/MTV-510 by updating section of 4.6, "Migration plan options" of the MTV user guide.

Preview: http://file.emea.redhat.com/rhoch/cleanup_plan_options/html-single/#running-migration-plan_mtv

### CODE REVIEW

**Comment by RichardHoch** [comment]
@ahadas @qiyuann Please review this PR.

File: documentation/modules/migration-plan-options-ui.adoc [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 On the *Plans for virtualization* page of the {ocp} web console, you can click the {kebab} beside a migration plan to access the following options:
 
+* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, you are prompted to download a `.tar` archive file.
```
Comment (qiyuann) [inline documentation/modules/migration-plan-options-ui.adoc:11]: This should be similar to the `Get logs` part in https://github.com/kubev2v/forklift-documentation/pull/384

```suggestion
* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, wait until *Get logs*  changes to *Download logs* and then click the button to download the logs.
```

**Comment by RichardHoch** [comment]
@qiyuann I made the change you requested. Please review the PR again.

File: documentation/modules/migration-plan-options-ui.adoc [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 On the *Plans for virtualization* page of the {ocp} web console, you can click the {kebab} beside a migration plan to access the following options:
 
+* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, wait until *Get logs*  changes to *Download logs* and then click the button to download the logs.
```
Comment (qiyuann) [inline documentation/modules/migration-plan-options-ui.adoc:11]: Remove a redundant space on the right of the last `Get logs`

```suggestion
* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, wait until *Get logs* changes to *Download logs* and then click the button to download the logs.
```

**Comment by RichardHoch** [comment]
@qiyuann I removed the extra space. Please review this PR again.

**Review by qiyuann — APPROVED** [review]

**Review by ahadas — APPROVED** [review]

**Pull request merged**